src/ring_theory/int/basic.lean

```diff
@@ -341,3 +341,54 @@ instance decidable_int : decidable_rel (λ a b : ℤ, (multiplicity a b).dom) :=
 λ a b, decidable_of_iff _ finite_int_iff.symm
 
 end multiplicity
+
+lemma induction_on_primes {P : nat → Prop} (h₀ : P 0) (h₁ : P 1)
```

Comment:
Also, this doesn't belong in this file, surely?

Reply:
(The current proof uses the `unique_factorization_monoid` instance that's defined earlier in this file.)

Reply:
> Also, this doesn't belong in this file, surely?

I searched, but I didn't find a better place for this lemma...
Putting it here seemed reasonable (despite the file being named "basic"), as the header reads : 
> This file collects results for the integers and natural numbers that use abstract algebra in
> their proofs or cases of ℕ and ℤ being examples of structures in abstract algebra.

Reply:
@semorrison Do you still have doubts ?

Reply:
Nope, all fine.

(Very sorry for the delay getting back to this, I don't reliably check github notifications so have been doing drive-by PR reviews... I'll try to pay attention. :-)

Reply:
(My doubt had been because this lemma was purely about `nat`, but the file name says `int`.)